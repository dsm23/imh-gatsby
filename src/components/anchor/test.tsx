/// <reference types="jest" />

import React from "react";
import { render } from "@testing-library/react";

import Anchor from ".";

const renderable = <Anchor href="/foo">This is an anchor tag </Anchor>;

describe(`Anchor`, () => {
  it(`it renders`, () => {
    const { baseElement } = render(renderable);

    expect(baseElement).toBeTruthy();
  });

  it(`matches snapshot`, () => {
    const { asFragment } = render(renderable);

    expect(asFragment()).toMatchSnapshot();
  });
});
