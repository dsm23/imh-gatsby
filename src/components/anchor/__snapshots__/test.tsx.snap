// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor matches snapshot 1`] = `
<DocumentFragment>
  <a
    class="css-fd0fo2"
    href="/foo"
  >
    This is an anchor tag 
  </a>
</DocumentFragment>
`;
